Client/src/Components/Search/SearchView.tsx

```diff
@@ -18,7 +18,7 @@ export default function SearchView(props: IProps) {
   const [foundDatasets, setFoundDatasets] = useState<IDatasetModel[]>([])
 
   // array of selected ids
-  const [selection, setSelection] = useState<SelectionChangeParams>(null)
+  const [selection, setSelection] = useState<SelectionChangeParams>({ rowIds: [] })
```

Comment:
how does the rowId's work here?

Reply:
so the SelectionChangeParams is a type from DataGrid material component
when on the search view new rows are being selected it is reflected in the selection. selection.rowIds will contain the ids of the rows that have been selected. the bug was caused by a null exception cause selection was null and it was trying to access rowIds.